cmd/kube-controller-manager/app/plugins_storage_minimal.go

```diff
@@ -0,0 +1,154 @@
+// +build skip_storage_plugins_requiring_vendor_api
+
+/*
+Copyright 2014 The Kubernetes Authors.
```

Comment:
Should the year be changed to 2021 since this is a new file?

Reply:
Done!